### PR TITLE
Issue 5001 - Fix next round of UI bugs:

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:

Addressing a series of bugs found by QE:

Bug 2016526 - LDAPI & Autobind save btn misbehaving
Bug 2016481 - Disabling Security leaves the pop-up window open
Bug 2016026 - Selecting existing certificate in Security Configuration crashes browser
Bug 2017402 - Adding several allowed SASL mechanisms does not behave correctly
Bug 2017411 - cockpit crashes because invalid SASL mapping regex was saved
Bug 2022117 - Cockpit UI: Editing an Objectclass name causes an error in Cockpit UI
Bug 2021194 - Searching "matching rules" in the "Schema" Tab crashes browser
Bug 2021591 - cockpit : audit and audit failure log enablement status is not persistent

relates: https://github.com/389ds/389-ds-base/issues/5001

Reviewed by: ?

### CODE REVIEW

File: ldap/schema/00core.ldif [hunk 1]
```diff
@@ -305,6 +305,11 @@ attributeTypes: ( 1.3.6.1.4.1.1466.101.120.14 NAME 'supportedSASLMechanisms'
   SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
   USAGE dSAOperation
   X-ORIGIN 'RFC 4512' )
+
+attributeTypes: ( 2.16.840.1.113730.3.1.2386 NAME 'availableSASLMechanisms'
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
+  USAGE dSAOperation
+  X-ORIGIN 'RFC 4512' )
```
Comment (tbordaz) [inline ldap/schema/00core.ldif]: also needs rhanana registration. X-origin may be 'RFC 4512 - related' as it is not described in 4512

**Review by tbordaz — COMMENTED** [review]
You have my ACK for the core-server part.

File: ldap/schema/00core.ldif (hunk 1 above)
Comment (mreynolds389) [reply ldap/schema/00core.ldif]: yeah I lost access to rhanana, need to open a help desk ticket.  And I will remove that RFC reference

**Comment by droideck** [comment]
I've built the package with the PR

```
[root@ci-vm-10-0-139-70 ds]# rpm -qa | grep 389-ds
389-ds-base-libs-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-snmp-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-debugsource-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-debuginfo-2.1.0-20211112git7b82f3089.fc34.x86_64
cockpit-389-ds-2.1.0-20211112git7b82f3089.fc34.noarch
389-ds-base-snmp-debuginfo-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-libs-debuginfo-2.1.0-20211112git7b82f3089.fc34.x86_64
389-ds-base-devel-2.1.0-20211112git7b82f3089.fc34.x86_64
```

But when I create (and first start) the instance, I get the error:

```
[12/Nov/2021:17:15:30.230176065 -0500] - ERR - slapi_entry_schema_check_ext - Entry "cn=schema" has unknown object class "ldapSubentry"
[12/Nov/2021:17:15:30.234508875 -0500] - ERR - str2entry_dupcheck - Entry has no dn
[12/Nov/2021:17:15:30.237372930 -0500] - ERR - dse_read_one_file - Parsing entry (lineno: 309) in file /usr/share/dirsrv/schema/00core.ldif failed.
[12/Nov/2021:17:15:30.240814130 -0500] - ERR - dse_read_one_file - Invalid section [attributeTypes: ( 2.16.840.1.113730.3.1.2386 NAME 'availableSASLMechanisms'
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
  USAGE dSAOperation
  X-ORIGIN '389 Directory Server' )
#
################################################################################
#
attributeTypes: ( 2.5.4.41 NAME 'name'
  EQUALITY caseIgnoreMatch
  SUBSTR caseIgnoreSubstringsMatch
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
  X-ORIGIN 'RFC 4519' )
#
################################################################################
#
attributeTypes: ( 2.5.4.15 NAME 'businessCategory'
  EQUALITY caseIgnoreMatch
  SUBSTR caseIgnoreSubstringsMatch
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
  X-ORIGIN 'RFC 4519' )
#
################################################################################
#
attributeTypes: ( 2.5.4.6 NAME ( 'c' 'countryName' )
  SUP name
  SYNTAX 1.3.6.1.4.1.1466.115.121.1.11
  SINGLE-VALUE
  X-ORIGIN 'RFC 4519'
  X-DEPRECATED 'countryName' )
#
################################################################################
#
 ...]
[12/Nov/2021:17:15:30.243924830 -0500] - ERR - setup_internal_backends - Please edit the file to correct the reported problems and then restart the server.
```

Do I miss something in my setup? Hm...

**Comment by mreynolds389** [comment]
@droideck - hmmm the install worked for me, but there was definitely a bug in the schema file.  Should be fixed now...

**Review by droideck — APPROVED** [review]
All checked! Looks good to me!

**Pull request merged**